packages/tools/devtools/devtools-core/src/CommonInterfaces.ts

```diff
@@ -45,3 +48,41 @@ export interface HasFluidObjectId {
 	 */
 	fluidObjectId: FluidObjectId;
 }
+
+/**
+ * Base interface used in message data for edits allowed by a particular Fluid object (DDS) via
+ * an enum.
+ *
+ * @public
+ */
+export enum EditType {
+	number = "number",
+	string = "string",
+	boolean = "boolean",
+}
+
+/**
+ * Base interface used in message data for edit events containing the allowed edits as an array of EditType.
+ *
+ * @internal
+ */
+export interface HasEditType {
```

Comment:
For these new interfaces, it would probably be good to double check that they are used in multiple places. If not, it might be better remove them and inline their properties into the single usage site.